doc/modules/cross_validation.rst

```diff
@@ -209,6 +209,16 @@ The following sections list utilities to generate indices
 that can be used to generate dataset splits according to different cross
 validation strategies.
 
+Cross-validation iterators for IID data.
+========================================
+
+Data is Independent Identically Distributed (IID) if the individual samples
+of the data are independent of each other and are drawn from an identical
+distribution.
```

Comment:
Pedantic: At least it is assumed for most theoretical results in machine learning. I think in practice this may be violated as often as not.